### DISCUSSION TITLE
Chrome or Firefox Extension using Deno Fresh

### DISCUSSION THREAD

**Discussion by ajinkyashukla** [freshframework/fresh#2247] [Ideas]
I was dabbling around with Chrome extensions and thought of building one with Fresh framework. It seems given the architecture of the Fresh, there is no `index.html`. There does not seem to be an easy way of getting Fresh to be used for Chrome extension development.

Has someone built a Chrome or Firefox extension using Fresh ?

Can the Fresh authors consider adding a feature so that Fresh can be easily used for Chrome or Firefox extension development ?

**Comment by marvinhagemeister** [comment]
It's very unlikely that we will focus on making browser extension development a target for Fresh anytime soon, unless a majority of users needs this. Reason is we already have our hands full with supporting Fresh's current use cases of building websites/apps, so we simply lack the time to focus on that.

That said we're happy to accept PRs from the community if somebody is willing to take on that work and figure it out on their own.
